pkg/restore/restore.go

```diff
@@ -63,14 +63,19 @@ type VolumeSnapshotterGetter interface {
 	GetVolumeSnapshotter(name string) (velero.VolumeSnapshotter, error)
 }
 
+type Data struct {
```

Comment:
wonder if this struct should be called `Request` and embed the `velerov1api.Restore` object, to follow the same model as the `pkgbackup.Request` struct?